applications/audio/broadcast_config_tool/src/main.c

```diff
@@ -86,6 +93,11 @@ static enum bt_audio_location stream_location[CONFIG_BT_ISO_MAX_BIG]
 					     [CONFIG_BT_BAP_BROADCAST_SRC_STREAM_COUNT] = {
 						     {{BT_AUDIO_LOCATION_MONO_AUDIO}}};
 
+static uint8_t lc3_streamer_idx[CONFIG_BT_ISO_MAX_BIG][CONFIG_BT_BAP_BROADCAST_SRC_SUBGROUP_COUNT]
+			   [CONFIG_BT_BAP_BROADCAST_SRC_STREAM_COUNT];
+
+#define LC3_STREAMER_INDEX_UNUSED 0xFF
+
```

Comment:
```suggestion
#define LC3_STREAMER_INDEX_UNUSED 0xFF

static uint8_t lc3_streamer_idx[CONFIG_BT_ISO_MAX_BIG][CONFIG_BT_BAP_BROADCAST_SRC_SUBGROUP_COUNT]
			   [CONFIG_BT_BAP_BROADCAST_SRC_STREAM_COUNT] = {LC3_STREAMER_INDEX_UNUSED};
```
Then you can remove the memset in main.